x-pack/plugins/cases/public/components/subtitle/index.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const Wrapper = styled.div`
+  ${({ theme }) => css`
+    margin-top: ${theme.eui.euiSizeS};
+
+    .siemSubtitle__item {
```

Comment:
there are several css classes starting with `siem`, could you please (carefully) do a find and replace with `cases`?

Reply:
Yep, donezo. Thanks for catching 😄!